src/components/tab-title/tab-title.scss

```diff
@@ -91,18 +102,40 @@
   }
 }
 
-.container--has-text {
+.content--has-text {
   @apply p-1;
 }
 
-.container--has-text .calcite-tab-title--icon.icon-start {
+.content--has-text .calcite-tab-title--icon.icon-start {
   margin-inline-end: theme("margin.2");
 }
 
-.container--has-text .calcite-tab-title--icon.icon-end {
+.content--has-text .calcite-tab-title--icon.icon-end {
   margin-inline-start: theme("margin.2");
 }
 
+@mixin tab-title-element-base() {
+  @apply transition-default
+      text-color-3
+      box-border
+      flex
+      cursor-pointer
+      self-stretch
+      h-full
+      appearance-none
+      overflow-hidden
+      border-none
+      justify-center
+      items-center;
+}
+
+.close-button {
+  @include tab-title-element-base;
```

Comment:
This is the only place where `tab-title-element-base` is used, so let's inline its contents here.